Plugins/Events/Events/FeatEvents.cpp

```diff
@@ -49,7 +69,9 @@ void FeatEvents::UseFeatHook
     Events::PushEventData("TARGET_POSITION_Y", pvTarget ? std::to_string(pvTarget->y) : "0.0");
     Events::PushEventData("TARGET_POSITION_Z", pvTarget ? std::to_string(pvTarget->z) : "0.0");
 
-    Events::SignalEvent(before ? "NWNX_ON_USE_FEAT_BEFORE" : "NWNX_ON_USE_FEAT_AFTER", thisPtr->m_idSelf);
+    Events::SignalEvent("NWNX_ON_USE_FEAT_AFTER", thisPtr->m_idSelf);   
```

Comment:
Do we want to signal the AFTER event if it was bypassed?

Reply:
It's signaled in BhaalM's Healer's Kit event and Liareth's [example](https://github.com/Liareth/unified/blob/ff715cc2acd889f21aaf86f6426393d6f98ef66e/Plugins/Events/Events/AssociateEvents.cpp) code from #16, so probably? :D